include/perfetto/tracing/track.h

```diff
@@ -231,6 +231,11 @@ class PERFETTO_EXPORT_COMPONENT NamedTrack : public Track {
     return NamedTrack(std::forward<TrackEventName>(name), id, parent);
   }
 
+  template <class TrackEventName>
+  static NamedTrack Global(TrackEventName name, uint64_t id = 0) {
```

Comment:
`TrackEventName&&`, this needs to be a forwarding reference.

(NamedTrack::ThreadScoped was broken as well, I had to send https://github.com/google/perfetto/pull/1879)

Can you also cover this in api_integrationtest.cc, please, just to make sure it compiles? (It would be broken now).